### DISCUSSION TITLE
Wellcome to take a look for another style of AI workflow , LOL

### DISCUSSION THREAD

**Discussion by youkpan** [langgenius/dify#3642] [General]
Dify workflow is great job !
by the way, Anyone want to take a look for another UI style of AI workflow ?
here is my website ,I build by myself: [https://ai.zyinfo.pro/wf](https://ai.zyinfo.pro/wf)
Wellcome to use , LOL

![AI workflow](https://github.com/langgenius/dify/assets/2157423/787d3146-5575-4b84-a548-fec79e8d0f84)

**Comment by win2333** [comment]
有点复杂
